### DISCUSSION TITLE
Lens parameter

### DISCUSSION THREAD

**Discussion by BenRoe** [shark92651/LaserParamsConverter#11] [Q&A] [answered]
Hello,
first thanks for making this useful software.
The lens parameter in the software is the focal length or the working area?

**Comment by shark92651** [comment, marked as answer]
The lens parameter is the working area/field size of the lens

**Reply by SoLongSidekick** [reply]
So the dot size? I have a Creality Falcon 2 40w, my dot size is ~.1mm in 40w mode and ~.8mm in 20w mode. Are these also their numbers for the lens parameter?

**Comment by shark92651** [comment]
I don’t do anything with dot size. Also, I am not that familiar with diode lasers, but you should be able to use the conversion of power and speed to move your library between a 20W and 40W diode laser. Select the “CO2 Gantry” laser type and there is no lens parameter required.
